inc/classes/class_auth.php

```diff
@@ -14,70 +14,70 @@ class auth {
      *
      * @var array
      */
-    public $auth = [];
+    private $auth = [];
```

Comment:
There is a reference to this in `index_module.inc.php` (L32)

Reply:
Oh, true. Fixed.